packages/scene-composer/src/components/panels/scene-settings/MatterportTagSync.tsx

```diff
@@ -40,40 +40,37 @@ export const MatterportTagSync: React.FC = () => {
 
   const { handleAddMatterportTag, handleUpdateMatterportTag, handleRemoveMatterportTag } = useMatterportTags();
   const { mattertagObserver, tagObserver } = useMatterportObserver();
+  const [syncTagStatus, setSyncTagStatus] = useState<'loading' | 'success' | 'error'>('loading');
```

Comment:
You can make them as type and pass the type here.

Reply:
I think it's simple enough to just be defined inline here, and it will do its job to fail the build when a wrong string is set as the status.

Reply:
Do we know why assumed only 3 status type here? I still think having its own type better.

Reply:
And also initial state is set to loading. When this state get's cleared if user hasn't clicked the SyncMatterport button? Why I am asking is you are setting it to loading state again inside doSync method, and outside as well. But Not sure when this get's cleared?